lambda/src/main/resources/db/migration/V16__drop_alldigital_from_transfer_agreement.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "TransferAgreement" DROP COLUMN "AllDigital";
```

Comment:
This reminds me: you might already have this step on your list, but remember to remove the column from the [test DB script](https://github.com/nationalarchives/tdr-consignment-api/blob/master/src/test/resources/scripts/init.sql). I'd really like to [remove that script](https://national-archives.atlassian.net/browse/TDR-150) but it would be quite a lot of work.